### ISSUE TITLE
Mouse clicking is blocked

### ISSUE THREAD

**Issue by BrontoRed** [Noitidart/MouseControl#9]
After installing MouseControl v2.0-unlisted on Firefox 49.0.2 (Windows 8.1 x64) something weird happens periodically: mouse just stops using click function (left/middle/right click) and scrolling, even outside of Firefox window. I can move mouse pointer, highlight objects on web pages, buttons in different windows, but can't press on them. Closing Firefox through taskmanager (using keyboard) bring everything back to normal.
Previous version works fine.

**Comment by Noitidart** [comment]
Wow very weird. If you get a chance - If it stops working, please try clicking each button. I'm thinking the "button-up" event didn't register - please let me know if this fixes it.

Thanks very much for this report.

**Comment by BrontoRed** [comment]
Yes, this is caused by "button-down" effect, as if i'm holding the button. It makes such processes as text selection on page, dragging bookmarks in submenus unstoppable (if I manage to press mouse button by chance).
What is interesting, it can stop by itself, as started.
Checked this several times (but not on a clean profile), installing new and old version of MouseControl. MouseControl v2.0-unlisted definitely causes this problem.

**Comment by Noitidart** [comment]
Thanks for this detailed test. I will look into this :)

**Comment by Noitidart** [comment]
By the way were you able to undo the "button-down" by releasing then pressing again and releasing again that button?

**Comment by BrontoRed** [comment]
I don't know, what causes this effect to stop. My standart action are: switch to another window (alt+tab), check if everything is stuck (can't use mouse), then back to Firefox, press several times left button, switch to another window, wait for a minute. And here we are - mouse works again (no cpu or mem load). If not - shut down Firefox, this brings mouse to life.

**Comment by Noitidart** [comment]
Hm I'm working on this having trouble reproducing. I'll get this one for sure. I might put in an emergency hotkey which will disable the addon the re-enable it.

**Comment by BrontoRed** [comment]
I think, I found the reason for this problem. I'm using [Folder Menu 3](http://foldermenu.sourceforge.net/) - a program, which allows you to quickly jump to your favorite folders in explorer. It opens through predefined mouse button combination (in my case i use double clicking of the middle mouse button [MMB]). The problem appears everytime after double clicking MMB in Firefox window and opening some folder that I need. After going back from Explorer to Firefox, "button-down" effect is present until I click MMB one or several times _or_ restart Firefox _or_ close Folder Menu 3.
This can be reproduced on clean Firefox profile (no other add-ons installed). Hope you can fix it.
